write_spectra/header_function.py

```diff
@@ -0,0 +1,160 @@
+from pandas import to_datetime
+from astropy.time import Time
+from datetime import date
+import astropy.io.fits as fits
+import numpy as np
+
+def compile_header(wavelength_data, **spectra_data_info):
+    """Creates a header from a dictionary of values. """
+
+    history1 = None
+    history2 = spectra_data_info['generated_history']
+
+    header = fits.Header()
+    header.set('EXTNAME', 'PRIMARY', 'name of this extension')
+
+    try:
+        header.set('VOCLASS', spectra_data_info['voclass'], 'VO Data Model')
+    except KeyError:
+        pass
+
+    # Target Data
+    try:
+        header.set('OBJECT', spectra_data_info['object_name'], 'Name of observed object')
+    except KeyError:
+        pass
+
+    try:
+        ra = spectra_data_info['RA']
+        header.set('RA', ra, '[deg] Pointing position')
+    except KeyError:
+        ra = None
+
+    try:
+        dec = spectra_data_info['dec']
+        header.set('DEC', dec, '[deg] Pointing position')
+    except KeyError:
+        dec = None
```

Comment:
Is this case sensitive? In other words, does it not find spectra_data_info['Dec']?